.travis.yml

```diff
@@ -6,17 +6,20 @@ rvm:
   - 2.3.8
   - 2.4.5
   - 2.5.3
+  - 2.6.4
 
 gemfile:
   - gemfiles/rails_4.1.gemfile
   - gemfiles/rails_4.2.gemfile
   - gemfiles/rails_5.0.gemfile
   - gemfiles/rails_5.1.gemfile
   - gemfiles/rails_5.2.gemfile
+  - gemfiles/rails_6.0.gemfile
 
 before_install:
 - gem update --system --no-doc
-- gem install bundler
+- gem install -v 1.17.3 bundler --no-rdoc --no-ri
+- bundle _1.17.3_ install
```

Comment:
We should remove this as Travis will run this command automatically with the specific version of bundler we have installed.
```suggestion
```